eda_analysis/eda_analysis.py

```diff
@@ -78,8 +85,50 @@ def describe_cat_var(dataframe,cat_vars):
     >>> describe_cat_variable(X,cat_vars)
        
     """
-    # Code 
     
+    #Checking for valid inputs
+    if not isinstance(dataframe, pd.DataFrame):
+        raise Exception("The value of the argument 'dataframe' must be of type 'pandas.DataFrame'")
+    
+    if not isinstance(cat_vars,list) or not all(isinstance(x, str) for x in cat_vars):
+        raise Exception("The value of the argument 'cat_vars' must be a list of strings")
+    
+    dataframe=dataframe.dropna()
+    col_set = set(dataframe.columns)
+    col_subset = set(cat_vars)
+    if not col_subset.issubset(col_set):
+        raise Exception("The input categorical column names must belong to the dataframe")
+    
+
+    data = dataframe[cat_vars]
```

Comment:
```suggestion
    col_set = set(dataframe.columns)
    col_subset = set(cat_vars)
    if not col_subset.issubset(col_set):
        raise Exception("The input categorical column names must belong to the dataframe")
    
    dataframe=dataframe.dropna()
    data = dataframe[col_subset]
```

Reply:
1. I think NA values should be dropped only after the dataframe is subsetted.
2. Use `col_subset` to subset if you do not require the elements in cat_vars to be unique. Otherwise, if they are not unique, you will get an error.